cms/djangoapps/contentstore/tasks.py

```diff
@@ -1507,6 +1507,6 @@ def handle_unlink_upstream_block(upstream_usage_key_string: str) -> None:
     ):
         make_copied_tags_editable(str(link.downstream_usage_key))
     for link in ContainerLink.objects.filter(
-        upstream_usage_key=upstream_usage_key,
+        upstream_container_key=upstream_usage_key,
```

Comment:
@navinkarkera Sorry, could I get some more context for this change? What's the issue it fixes?

Reply:
Sorry @navinkarkera , I still don't understand.. Upstream container keys aren't UsageKeys, so won't we get a InvalidKeyError above if this function is called with a container key?

Reply:
@pomegranited You are completely right! Moved this part to its own task when containers are deleted.